.github/workflows/gradle-pr-build.yml

```diff
@@ -35,10 +35,3 @@ jobs:
         run: ./gradlew build ktlintCheck
       - name: Build with dev Kotlin version
         run: ./gradlew -PkotlinDev build ktlintCheck
-      - name: Upload test results
-        uses: actions/upload-artifact@v3
-        if: failure()
-        with:
-          name: Test artifacts
-          path: '**/build/reports/tests/**'
-          if-no-files-found: warn
```

Comment:
I'm removing this action, you can view test results via Gradle build scan.

Reply:
I am not liking this. I don't like to fill in my email adress each time that I want to get access to a build scan. I am developing using a laptop from my employer. I don't want to use my work email adress to get access to the scan. Neither do I want to access my personal email on my work laptop.

What would be the benefit of removing it?

Reply:
It's easier to view and share build scans, I noticed I have no need to fill email address to view build scans that had been published on CI, you can have a try like https://gradle.com/s/tcalcngp6a3tk.

Reply:
Yes, I just noticed that too in another failed build. And in no way, I meant that the Gradle build scan had to be removed ;-)
So let try it out then.